.github/workflows/deploy-workflow.yml

```diff
@@ -165,3 +165,23 @@ jobs:
           --set global.sql.datasource.host=${POSTGRES_HOST} \
           --set global.sql.datasource.hostForpostgresqlClient=${POSTGRES_CLIENT_HOST} \
           --set global.sql.datasource.url=${POSTGRES_URL}
+
+      - name: install datahub-monitoring helm
+        shell: bash
+        env:
+          KUBE_NAMESPACE: ${{ secrets.kube_namespace }}
+          DASHBOARD_NAME: datahub-status-dashboard
+        run: |
+          envsubst < helm_deploy/monitoring/datahub-networkpolicy.yaml \
+          | kubectl apply -f - --namespace=${KUBE_NAMESPACE}
+
+          kubectl create configmap ${DASHBOARD_NAME} \
```

Comment:
very minor but this might be a bit more readable if you split it up into separate commands rather than pipe it through (I initially confused it with the yaml pipes)

Reply:
I initially had the pipes indent, then removed it. Would that be helpful? 

Pipes were used as [there's not a particularly elegant way to `label` and `apply` via kubectl](https://github.com/kubernetes/kubernetes/issues/60295#issuecomment-552750699) (`apply` is declarative, and thus preferred over `create` commands which are imperative)